js/friday.js

```diff
@@ -1,34 +1,38 @@
 (function IsItFridayYet() {
   const date = new Date();
-  const theMonth = date.getMonth();
-  const theDay = date.getDay();
-  const element = document.getElementById("friday");
-  const spans = document.getElementsByClassName("fallingElement");
+  const thisMonth = date.getMonth();
+  const thisDay = date.getDay();
+  const theBody = document.body;
+  const textElement = document.getElementById("friday");
   let season = "";
 
```

Comment:
I want to have a long and exhausting conversation about Pure Functions (tm). The concept of Pure Functions (tm) is that we as JS developers have had a real shit habit of making overly complex functions that may result in side-effects. In the world of React, Angular, Vue and the like, We can no longer afford to produce verbose functions. In a Functional Programming methodology, the core tenant is simple and pure writing of functions that depend on nothing and should only focus on the mastery of its only command.

Let us really dive into the goal of these edits.
1. Create a function that only tells me if it's Friday or not based on a number being passed (my example below can be further improved but for now, pretend I did this thing real good).
2. Create a function that only tells me hat season we are currently.
3. Create a function that just changes site styles
4. Make them all have a failsafe or at least not break the site.
5. Daisy chain the invocation of the functions if it makes sense to do so.

Though the suggestions in this PR can still be refined, this is a good jumping-off point to writing bite-sized, hyperspecific Pure Functions...

...

...(tm)